docs/serving/app-security/security-guard-quickstart.md

```diff
@@ -0,0 +1,130 @@
+            queue-sidecar-image: ghcr.io/knative.dev/security-guard/guard-service
+        ```
+
+        If you are not using an operator, you may use the following command to update teh config-deployment ConfigMap:
+        ```
+        kubectl apply -f https://raw.githubusercontent.com/knative-sandbox/security-guard/main/deploy/queue-proxy.yaml`
+        ```
+
+    1. Add the necessary Guard resources to your cluster using:
+
+        ```
+        kubectl apply -f https://raw.githubusercontent.com/knative-sandbox/security-guard/main/config/resources/gateAccount.yaml
+        kubectl apply -f https://raw.githubusercontent.com/knative-sandbox/security-guard/main/config/resources/serviceAccount.yaml
+        kubectl apply -f https://raw.githubusercontent.com/knative-sandbox/security-guard/main/config/resources/guardiansCrd.yaml
+        ```
+
+    1. Deploy the gate-service on your system to enable automated learning of micro-rules.
+
+        ```
+        kubectl apply -f https://raw.githubusercontent.com/knative-sandbox/security-guard/main/deploy/guard-service.yaml`
+        ```
+
+## Creating and deploying a service
+
+!!! tip
+
+    The following commands create a `helloworld-go` sample Service while activating and configuring the `security-guard` extension for this Service. You can modify these commands, including changing the `security-guard` configuration for your service using either the `kn` CLI or changing the service yaml based on this example.
+
+Create a sample securedService:
+
+=== "Apply YAML"
+
+    1. Create a YAML file using the following example:
+
+        ```yaml
+        apiVersion: serving.knative.dev/v1
+        kind: Service
+        metadata:
+          name: helloworld-go
+          namespace: default
+        spec:
+          template:
+            metadata:
+                annotations:
+                  features.knative.dev/queueproxy-podinfo: enabled
+                  qpoption.knative.dev/guard-activate: enable
+            spec:
+              containers:
+                - image: gcr.io/knative-samples/helloworld-go
+                  env:
+                    - name: TARGET
+                      value: "Secured World"
+        ```
+
+    1. Apply the YAML file by running the command:
+
+        ```bash
+        kubectl apply -f <filename>.yaml
+        ```
+        Where `<filename>` is the name of the file you created in the previous step.
+
+=== "kn CLI"
+
+    ```
+    kn service create helloworld-go \
+        --image gcr.io/knative-samples/helloworld-go \
+        --env "TARGET=Secured World" \
+        --annotation features.knative.dev/queueproxy-podinfo=enabled \
+        --annotation qpoption.knative.dev/guard-activate=enable
+    ```
+
+After the Service has been created, Guard starts monitoring the Service Pods and all Events sent to the Service.
+
+## Managing the security of the service
+
+Guard offers situational awareness by writing its alerts to the Service queue proxy log.
+
+Example output:
```

Comment:
It'd be nice to include an example of how the user could generate an alert log themselves, so they could see it in action

Reply:
I followed the instructions as written here, and when I curled the service the only log I see in QP is a "Starting queue-proxy" one